File: TEO/Mav.UI/Mav.Ui/src/app/models/search/user.assessment.section.listing.model.ts
export class UserAssessmentSectionListingModel {

    public sectionID: number;

    public userAssessmentSectionID?: number;

    public number: number;

    public title: string;

    public statusID: number;

    public totalScore: number;

    public totalQuestions: number;

    public attempts: number;

    public actionAllowed: boolean;

    public showStatement: boolean;

    public maxAttempts?: number;

    public isEoQualification?: boolean;

    public timeRemaining?: number;
}

File: TEO/Mav.Models/SearchModels/UserAssessmentSectionListingModel.cs
using Mav.Data.Entities;
using Mav.Models.ViewModels;

namespace Mav.Models.SearchModels
{
    public class UserAssessmentSectionListingModel
    {
        public int SectionID { get; set; }

        public int StatementID { get; set; }

        public int? UserAssessmentSectionID { get; set; }

        public int Number { get; set; }

        public string Title { get; set; }

        public int StatusID { get; set; }

        public int TotalScore { get; set; }

        public int TotalQuestions { get; set; }

        public int Attempts { get; set; }

        public bool ActionAllowed { get; set; }

        public int MaxAttempts { get; set; }

        public bool? IsEoQualification { get; set; }

        public double? TimeRemaining { get; set; }
    }
}
